dev_apps/music-nga/elements/music-tab-bar.js

```diff
@@ -39,6 +42,7 @@ proto.createdCallback = function() {
   });
 
   this.selectedIndex = 0;
+  setTimeout(() => this.classList.add('resolved'), 0);
```

Comment:
nit: We don't need the `0` argument -- Also, why not use `requestAnimationFrame()` instead?